pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md

```diff
@@ -11,29 +11,29 @@ updated: 2024-01-22
 
 ## Objective
 
-**This page provides an overview of the lifecycle policy of VMware on OVHcloud services.**
+**This page provides an overview of the lifecycle policy of Hosted Private Cloud VMware on OVHcloud services.**
 
-OVHcloud's Hosted Private Cloud VMware service proposes dedicated infrastructure based on VMware SDDC technologies as well as other editors' technologies (such as Veeam or Zerto).
+Hosted Private Cloud VMware on OVHcloud services proposes dedicated infrastructure based on SDDC technologies as well as other editors' technologies (such as Veeam or Zerto).
 
-OVHcloud has a lifecycle policy for the service in order to take into account several factors, such as:
+OVHcloud has a lifecycle policy for the service in order to take into account several factors, such as :
```

Comment:
There is no space in English before punctuation signs. Please revert this change on lines 18, 21, 22, 129 for all guides except fr-fr and fr-ca.